novawallet/Modules/NovaMainAppContainer/NovaMainAppContainerViewController.swift

```diff
@@ -120,6 +135,36 @@ private extension NovaMainAppContainerViewController {
         )
     }
 
+    func findCollectionView(in view: UIView) -> UICollectionView? {
```

Comment:
I would move that 2 functions to the new protocol extension (Lets say `CollectionViewRotationHandling`) just to emphasise the reason we are adding them